assignments/arrays.js

```diff
@@ -63,34 +63,118 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+//console.log(inventory[32]);
+//console.log(Object.values(inventory[32]));
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );
+//console.log(inventory[32].car_year );
+//console.log(inventory[32].car_make );
+//console.log(inventory[32].car_model );
+//console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
+
+
 let lastCar = 0;
-console.log();
+//let lastCar = inventory[inventory.length-1];
+//console.log(inventory[inventory.length-1].car_make );
+//console.log(inventory[inventory.length-1].car_model );
+let finCar = `Last car is a ${inventory[inventory.length-1].car_make} ${inventory[32].car_model}`
+
+
+console.log(finCar);
 
 // ==== Challenge 3 ====
```

Comment:
give this one another shot. 

You're returning way too much info in this one. We just want the models in alphabetical order but instead we're seeing entire objects returned in this array. There is an easier way to knock this out.